modules/tests/test_food.py

```diff
@@ -0,0 +1,8 @@
+import modules
+
+
+def test_food():
+    assert ('food' == modules.process_query('Is Giordano\'s pizza in Chicago good?')[0])
+    assert ('food' == modules.process_query('Chipotle in Evanston')[0])
+    assert ('food' == modules.process_query('What are some restaurants in Chicago')[0])
+    assert ('food' != modules.process_query('something random')[0])
```

Comment:
Let's add blank line at the end the final one :)

Reply:
I've added blank lines to the ends of the two files but github's just not displaying them in the preview windows but they're there